guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -1,10 +1,10 @@
 [id="Adding_a_Baseboard_Management_Controller_Interface_{context}"]
 = Adding a baseboard management controller (BMC) interface
 
+{Project} can control the power status of bare metal hosts.
```

Comment:
```suggestion
{Project} can control the power status of bare-metal hosts.
```

Reply:
Applied and created a commit to fix all other instances.